lua/lazyvim/plugins/extras/lang/typst.lua

```diff
@@ -0,0 +1,48 @@
+return {
+  recommended = function()
+    return LazyVim.extras.wants({
+      ft = { "typst" },
+    })
+  end,
+
```

Comment:
I think these lines are unnecessary, as `tinymist` should install automatically when added to `opts.servers` in `nvim-lspconfig`.

https://github.com/LazyVim/LazyVim/blob/a66b44a9fe0031c85a536f13f84582a6ba7c3f82/lua/lazyvim/plugins/lsp/init.lua#L222-L246

Reply:
Yes, that is unnecessary. If defined in `nvim-lspconfig` it will install automatically. That check is being made if the server exists in `all_mslp_servers` or not. If it exists it will be added into `ensure_installed` table, otherwise if it doesn't exist it will just call the setup function assuming that you installed via other means.